src/Services/privateApiService.js

```diff
@@ -13,4 +13,16 @@ const Get = () => {
     .catch((err) => console.log(err));
 };
 
+const tokenVer = () => {
+  if (window.localStorage.hasOwnProperty('token')) {
```

Comment:
instead of if(...) and all the logic, you can make
```
if(!window.localStorage.hasOwnProperty('token')) return new Error('no token found');
const token = localStorage.token;

const Header = {
      Authorization: 'Bearer' + token,
 };

return Header;

```